### PR TITLE
OS Login staging updates

### PR DESCRIPTION
**Pull request by zmarano**

### CODE REVIEW

File: daisy_workflows/build-publish/partner/cos_81_lts.publish.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Name": "cos-81-lts",
+  {{if eq .environment "oslogin-staging" -}}
+  "WorkProject": "oslogin-staging-images",
```
Comment (illfelder) [inline daisy_workflows/build-publish/partner/cos_81_lts.publish.json:4]: Shouldn't this be oslogin-staging-project? Or did you set up a new project?

Comment (zmarano) [reply daisy_workflows/build-publish/partner/cos_81_lts.publish.json:4]: No, this is just the env variable in the template. The publish templates are using oslogin-staging-project.

Comment (illfelder) [reply daisy_workflows/build-publish/partner/cos_81_lts.publish.json:4]: Thanks!

Comment (zmarano) [reply daisy_workflows/build-publish/partner/cos_81_lts.publish.json]: Oh yeah ok sorry- wrong project :)

**Review by illfelder — APPROVED** [review]

**Pull request merged**